mx.truffleruby/suite.py

```diff
@@ -202,6 +202,7 @@
                 "java.logging",
                 "java.management",
                 "jdk.unsupported", # sun.misc.Signal
+                "jdk.management",
```

Comment:
Could you move it one line above? So it remains sorted.

Reply:
Done.